game/state/city/economyinfo.cpp

```diff
@@ -45,6 +45,7 @@ bool EconomyInfo::update(GameState &state, bool xcom)
 		{
 			currentPrice = currentPrice * randBoundsInclusive(state.rng, 95, 97);
 		}
+		currentPrice = std::lround(static_cast<double>(currentPrice) / 100.0);
```

Comment:
If this 'currentPrice' is actually 100* the price (to avoid floating point weirdness I guess?), please name it something different or to the /100 immediately - as otherwise I can see someone trying to modify this in the future getting confused.

Reply:
The `currentPrice` is a member of `EconomyInfo` i.e the actual price of the item. The /100 part was missing so I added it to compensate since `randBoundsInclusive` only works with integer values.
I just changed that in 01bb029 to use uniform_real_distribution, otherwise `randBoundsInclusive` would only allow increases/decreases in price by whole percents like 1%, 2%, 3%, etc